bundler/spec/commands/info_spec.rb

```diff
@@ -125,6 +125,22 @@
     end
   end
 
+  context "when gem has a reverse dependency" do
+    it "prints the details" do
+      bundle "info actionpack"
+
+      expect(out).to include("Reverse Dependencies:\n\t\trails (2.3.2) depends on actionpack (= 2.3.2)")
+    end
+  end
+
+  context "when gem has no reverse dependencies" do
+    it "excludes the reverse dependencies field from the output" do
+      bundle "info rails"
+
+      expect(out).not_to include("Reverse Dependencies:")
+    end
+  end
```

Comment:
I think you probably need to nest these specs so that the main `before` hooks to setup the sample application and install the `Gemfile` runs first.

Reply:
@deivid-rodriguez you were right, I think that fixed them.

We are still seeing some failures, but for a different reason: `Error: The operation was canceled.`  Looks like there might be some sort of temporary problem with Actions timing out.  I'll re-run the tests in a day or two to see if those clear up.

Reply:
@deivid-rodriguez after I rebased to trigger a re-run of the tests, we're still seeing some failures that seem unrelated to this PR.  I'm stumped.  Any ideas what's going on?

[For example](https://github.com/rubygems/rubygems/runs/4692650173?check_suite_focus=true):
```
Bundler::GemHelper gem management #build_checksum calculates SHA512 of the content
     Failure/Error: expect(File.binread(sha_path)).to eql(Digest::SHA512.hexdigest(""))
```

Reply:
Mmm you linked to a failure in a different PR, can you double check the link?

Reply:
I had not yet approved your latest modifications so maybe CI had not run yet. Let's see how it goes now.

Reply:
@deivid-rodriguez one of the test expectations was missing a space and I was only running `rake test` so I wasn't catching it locally.  Fixed now if you'd like to approve the latest modifications.  Sorry for all the back-and-forth here.

Reply:
@deivid-rodriguez friendly ping.  Hopefully this is ready for merge 🙂

Reply:
Thanks, and sorry I lost track of this one, I will review it soon.